public-ui/src/app/data-browser/charts/chart-age/chart-age-react.component.tsx

```diff
@@ -56,6 +63,129 @@ export class AgeChartReactComponent extends React.Component<Props, State> {
+      }
+      baseOptions.series = series;
+  }
+
+  prepEhrOrSurveyCategoriesAndData(ageAnalysisResults) {
+    const {ageAnalysis: {analysisId}, ageCountAnalysis: {results}, domain} = this.props;
+    const data = [];
+    const cats = [];
+    const color = '#2691D0';
+    let seriesName = '';
+    if (domain === 'ehr') {
+        seriesName = 'Age at First Occurrence in Participant Record';
+    } else if (domain === 'survey') {
+        seriesName = 'Age When Survey Was Taken';
+    } else {
+        seriesName = 'Age When Physical Measurement Was Taken';
+    }
+    // LOOP CREATES DYNAMIC CHART VARS
+    for (const a of ageAnalysisResults) {
+          // For normal Gender Analysis , the stratum2 is the gender . For ppi it is stratum5;
+          const ageResult = results.filter(x => (domain === 'survey' ? x.stratum2 : x.stratum4) ===
+          (domain === 'survey' ? a.stratum5 : a.stratum2))[0];
+          const count = (a.countValue <= 20) ? '&le; 20' : a.countValue;
+          const totalCount = (ageResult.countValue <= 20) ? '&le; 20'
+          : ageResult.countValue.toLocaleString();
+          if (a.analysisStratumName === null) {
+           a.analysisStratumName = AGE_STRATUM_MAP[(domain === 'ehr' ? a.stratum2 : a.stratum5)];
+          }
+          const analysisStratumName = a.analysisStratumName;
+          const percentage = Number(((a.countValue / ageResult.countValue) * 100).toFixed());
+          data.push({
+            name: a.analysisStratumName,
+            y: a.countValue,
+            color: color,
+            sliced: true,
+            toolTipHelpText: this.getTooltipHelpText(count, analysisStratumName, percentage,
+            totalCount, domain),
+            medicalConceptPercentage: percentage,
+            analysisId: analysisId
+          });
+          cats.push(a.analysisStratumName);
+        }
+        data.sort((a, b) => a.name.localeCompare(b.name));
+        cats.sort();
+        const dataOnlyLT20 = data.filter(x => x.y > 20).length === 0;
+        const series = [
+          {
+          color: color,
+          legendColor: color,
+          name: seriesName + ', Selected Answered Count',
+          colorByPoint: false,
+          data: data,
+          dataOnlyLT20: dataOnlyLT20
+          }];
+        return { categories: cats, series: series};
+  }
+
+  getTooltipHelpText(count, analysisStratumName, percentage, totalCount, domain) {
+      const toolTipHelpText = '<div class="chart-tooltip">' +
+                      '<strong>' + count + '</strong>' + ' participants were ages within range ' +
+                      analysisStratumName + ' when' + (domain === 'pm' ? ' physical measurement with' : '') +
+                      ' this medical concept first occurred and that is <strong>' +
+                      percentage + '</strong>' + '% of all participants with the same criteria. (Total Count = <strong> '
+                      + totalCount + '</strong>) </div>';
+      return toolTipHelpText;
```

Comment:
Setting to a const is redundant here, can just return the template:
```suggestion
      return '<div class="chart-tooltip">' +
                      '<strong>' + count + '</strong>' + ' participants were ages within range ' +
                      analysisStratumName + ' when' + (domain === 'pm' ? ' physical measurement with' : '') +
                      ' this medical concept first occurred and that is <strong>' +
                      percentage + '</strong>' + '% of all participants with the same criteria. (Total Count = <strong> '
                      + totalCount + '</strong>) </div>';
```

Reply:
Fixed thanks.